extensions/app-provider/pkg/config/config.go

```diff
@@ -68,7 +68,7 @@ type WOPIDriver struct {
 	AppInternalURL string `yaml:"app_internal_url" env:"APP_PROVIDER_WOPI_APP_INTERNAL_URL" desc:"internal url to the app, eg in your DMZ"`
 	AppName        string `yaml:"app_name" env:"APP_PROVIDER_WOPI_APP_NAME" desc:"human readable app name"`
 	AppURL         string `yaml:"app_url" env:"APP_PROVIDER_WOPI_APP_URL" desc:"url for end users to access the app"`
-	Insecure       bool   `yaml:"insecure" env:"APP_PROVIDER_WOPI_INSECURE" desc:"allow insecure connections to the app"`
+	Insecure       bool   `yaml:"insecure" env:"OCIS_INSECURE;APP_PROVIDER_WOPI_INSECURE" desc:"allow insecure connections to the app"`
```

Comment:
@wkloucek or was this left out intentionally?

Reply:
this is not about oCIS using invalid certificates, but instead the CS3org WOPI server. Therefore we shouln't use `OCIS_INSECURE` here